src/include/duckdb/main/relation.hpp

```diff
@@ -81,9 +84,12 @@ class Relation : public std::enable_shared_from_this<Relation> {
 	DUCKDB_API shared_ptr<Relation> Project(const string &select_list, const vector<string> &aliases);
 	DUCKDB_API shared_ptr<Relation> Project(const vector<string> &expressions);
 	DUCKDB_API shared_ptr<Relation> Project(const vector<string> &expressions, const vector<string> &aliases);
+	DUCKDB_API shared_ptr<Relation> Select(vector<unique_ptr<ParsedExpression>> expressions,
```

Comment:
Might be better if it's just called Project as well, since it does the same thing, the parameters are just different